dask_cloudprovider/tests/test_imports.py

```diff
@@ -23,3 +24,5 @@ def test_import_exceptions():
         from dask_cloudprovider import GCPCluster  # noqa
     with pytest.raises(ImportError):
         from dask_cloudprovider import DropletCluster  # noqa
+    with pytest.raises(ImportError):
+        from dask_cloudprovider import HetznerCluster  # noqa
```

Comment:
Please remove this, it is not necessary as this is a new cluster manager.

Reply:
done!